css/posts.scss

```diff
@@ -61,4 +70,45 @@ $preview-height: 150px;
   * {
     color: black;
   }
-}
\ No newline at end of file
+}
+
+.post-header{
+  width:100%;
+  height: auto;
+  background: linear-gradient(to bottom, rgba(0,0,0,0) 100%, white 100%);
+  margin-bottom: -5px;
```

Comment:
Negative marigns can cause issues later. This should probably be translated to a positive `margin-top` for the main body.

Reply:
There was a weird gap in the div, but not in the image. Changing this didn't seem to fix it